tests/unit/test_tiler.py

```diff
@@ -777,6 +778,53 @@ def it_knows_whether_coordinates_are_within_extraction_mask(
         assert type(coords_within_extraction_mask) == bool
         assert coords_within_extraction_mask == expected_result
 
+    @pytest.mark.parametrize(
+        "alpha, pass_tiles, outline, expected_topleft",
+        (
+            (128, False, "red", (255, 255, 155, 155)),
+            (None, True, [[0.1, 0.2, 0.3], [0.4, 0.5, 0.6]], (102, 102, 155, 155)),
+        ),
+    )
+    def it_can_locate_tiles(
```

Comment:
I can buy this unit test, but given the several changes I'd like to see at least 1 integration test for the edge cases used here

Reply:
Actually, the edge cases are best resolved with a unit test. I added a unit test specifically for the edge cases. I also added integration test cases to make sure the new outline functionality produces the expected output.

Reply:
ok nice